Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
+  override GOGCFLAGS += -N -l
```

Comment:
Why these changes? 
Without the quotes around -N -l, make all DEBUG=1 fails.
Without the else case to set GOGCFLAGS to "" make all fails.
Without the DEBUG ?= 0 `DEBUG=` is passed to the make command.